tests-runner/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM buildpack-deps:bookworm
+
+WORKDIR /tests-runner
+
+RUN apt-get update && apt-get install -y protobuf-compiler gcc-x86-64-linux-gnu curl bison npm python3-pip bash bsdmainutils
+
+# required for gvm
+SHELL ["/bin/bash", "-c"]
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o rustup-init.sh
+RUN chmod +x rustup-init.sh
+RUN ./rustup-init.sh -y -c rustfmt --default-toolchain nightly-2023-04-19
```

Comment:
Is `clippy` included by default?

Reply:
I don't think we need clippy for the e2e tests?